shared/router-v2/tab-bar/index.desktop.tsx

```diff
@@ -172,6 +184,7 @@ class TabBar extends React.PureComponent<Props, State> {
             </Kb.ClickableBox>
           ))}
           <RuntimeStats />
+          <LoadFsStuff />
```

Comment:
did you talk to Nojima/Mike about adding additional stuff that runs at login? or is that only an issue on mobile?

Reply:
Not yet; planning to!

Reply:
I ended up getting some free help from strib to have some Go sweetness to power this badge in a cheap way. So will move these back to the Fs tab.

Reply:
Woo free help from strib is the best